examples/hello-world/src/App.js

```diff
@@ -14,16 +14,19 @@ class App extends Component {
         const {name, unreadCount} = this.state;
 
         return (
-            <p>
-                <FormattedMessage
-                    id="welcome"
-                    defaultMessage={`Hello {name}, you have {unreadCount, number} {unreadCount, plural,
+            <div>
+                <p>
+                    <FormattedMessage
+                        id="welcome"
+                        defaultMessage={`Hello {name}, you have {unreadCount, number} {unreadCount, plural,
                       one {message}
                       other {messages}
                     }`}
-                    values={{name: <b>{name}</b>, unreadCount}}
-                />
-            </p>
+                        values={{ name: <b>{name}</b>, unreadCount }}
+                        />
+                </p>
+                <p> Last update <FormattedRelative value={Date.now() - 3600000} /> </p>
```

Comment:
Using `Date.now()` inside `render()` is an anti-pattern that I don't what to encourage in the examples.